Microsoft.Toolkit.Uwp.UI.Controls.Layout/BladeView/BladeView.xaml

```diff
@@ -156,8 +156,7 @@
                                     Background="{TemplateBinding CloseButtonBackground}"
                                     Style="{StaticResource ButtonRevealStyle}"
                                     BorderThickness="1"
-                                    Width="32"
-                                    Height="32"
+                                    VerticalAlignment="Stretch"
                                     Content="&#xE73F;"
                                     FontFamily="Segoe MDL2 Assets"
```

Comment:
@michael-hawker @XAML-Knight, let me know what you think about solving the problem like this. I figured if the button changes underneath us again this would handle it best. Another option was to just make the width bigger based on the increased padding.